kubernetes/launch-match.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash -e
+
+####################
+# Argument parsing #
+####################
+
+DEFAULT_NUM_GPUS=1
```

Comment:
defaulting to 1 GPU because if `--games` is not set, then the number of games ran will be `[numGamesTotal from match-base.cfg] * [num GPUs]`, which may be more than expected by the user